torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -5712,6 +5713,7 @@ def test_ddp_profiling_autograd_profiler(self):
             IS_MACOS or IS_WINDOWS,
             "torch.profiler not enabled for mac/windows: https://github.com/pytorch/pytorch/pull/56124",
         )
+        @unittest.skipIf(TEST_WITH_ROCM, "Skipping as we won't be fixing this test failure on an older version of pytorch")
```

Comment:
@rraminen You missed an import unittest in this file. Without that, we get a name 'unittest' is not defined error. Fixed in https://github.com/ROCm/pytorch/commit/dcbc5703cea53af17c9e7cdcf6a00b5be0589044